helm/node-health-chart/templates/pull-secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+type: kubernetes.io/dockerconfigjson
+metadata:
+  name: node-health-pull-secret
+  namespace: {{ .Values.namespace }}
+data:
+  .dockerconfigjson: {{ .Values.Installation.V1.Secret.Registry.PullSecret.DockerConfigJSON | b64enc | quote }}
```

Comment:
Chart-operator doesn't know about the installations repo, so I think when this service should be managed by chart-operator, this doesn't work. Also as mentioned above, I think you don't need a Pull Secret here after all.

Reply:
This is spot on. The installations templating won't work with chart-operator.

It can be done by creating the values configmap in cluster-operator and using a flag. A chartconfig CR will also need to be created in cluster-operator.

Reply:
Yes if the image is public then removing this pull secret is going to be the best idea IMO. The secrets support for chart-operator isn't there yet.